### ISSUE TITLE
[feat]: Add alpha and lr to hyperparameter search

### ISSUE THREAD

**Issue by a-lemus96** [a-lemus96/fs-nerf#9]
Implement the following changes to include alpha and learning rate values to the hyperparameter search:
- Split learning rate parameter into lro and lrf values
- Rename alpha parameter to ao
- Create separate YAML config files for L1 and L2 hyperparameter search
